src/Interfaces/LazySet.jl

```diff
@@ -67,8 +67,13 @@ julia> subtypes(LazySet, false)
 If we only consider *concrete* subtypes, then:
 
 ```jldoctest; setup = :(using LazySets: subtypes)
-julia> subtypes(LazySet, true)
-37-element Array{Type,1}:
+julia> concrete_subtypes = subtypes(LazySet, true);
+
+julia> length(concrete_subtypes)
+38
+
+julia> println.(concrete_subtypes);
+ AffineMap
```

Comment:
```suggestion
AffineMap
```
`println` does not add a leading space, so those need to be removed everywhere.